sdk/nodejs/storage/zMixins.ts

```diff
@@ -248,6 +250,34 @@ export class BlobEventSubscription extends appservice.EventSubscription<BlobCont
     }
 }
 
+export interface TableOutputBindingDefinition extends appservice.BindingDefinition {
+    /**
+    * The name of the property in the context object to bind the actual table output to.
+    */
+    name: string;
+
+    /**
+     * The name of the table the output should write to
+     */
+    tableName: pulumi.Input<string>;
+
+    /**
+     * The type of the table binding.  Must be 'table'.
+     */
+    type: "table";
+
+    /**
+     * The direction of the binding.  We only 'support' tables as outputs of functions.
+     */
+    direction: "out";
+

```

Comment:
extra newline.

Reply:
fixed in d783c138f08bff57325be4e7fb98e3a6933d5b8b